src/PiWeb.Import.Sdk/ImportData/Entity.cs

```diff
@@ -0,0 +1,296 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+#region usings
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.ImportData;
+
+/// <summary>
+/// Represents an entity to be imported.
+/// </summary>
+public abstract class Entity(Guid uuid) : IExtendedAttributeDictionary, IVariableDictionary, IAdditionalDataDictionary
+{
+    #region members
+
+    private EmbeddedValueDictionary<ushort, AttributeValue, StaticUShortEqualityComparer> _Attributes =
+        new EmbeddedValueDictionary<ushort, AttributeValue, StaticUShortEqualityComparer>();
+
+    private EmbeddedValueDictionary<string, VariableValue, StaticOrdinalStringEqualityComparer> _Variables =

```

Comment:
The comparer is case-sensitive while the old TreeDataImage is using a case-insensitive comparer.
Do we want to be more restrictive in the future or not?